src/store/reducers/teams.js

```diff
@@ -0,0 +1,37 @@
+import {
+  GET_TEAMS_SUCCESS,
+  GET_TEAMS_FAILED,
+  ADD_USER_TEAMS_FAILED,
+  ADD_USER_TEAMS_SUCCESS,
+} from '../actions/types';
+
+const initialState = {
+  teams: [],
```

Comment:
You suggest to only use the getTeams service's fonction in ListTeams for exemple ? And not stock the data in the store ? I stocked it in the store to avoid multiple api calls for exemple, when editing a team, you don't need to call GET /teams/:id in the api, you just need to filter on the whole array. Maybe we should add a route to get only our teams, not all teams. But I don't understand why using a store if it's not to stock the data ! But I maybe missed a point 😅

Reply:
But maybe it's not used so often that it needs to be stored in the store, especially for all users?

Reply:
@remirubis

Reply:
It's not necessarily a problem to fetch everything but it's unmanageable when you start to have thousands of projects, the load would be too important and it would greatly increase the loading time when you load the page for the first time, and your implementation fetches every time we arrive on the page /teams all the teams so in fact no real interest to store them. Concerning the update it would be too dangerous to base on the store to start modifications if another user made modifications since. One of the solutions would be to put only the teams of the connected user which will greatly reduce the load